src/routes/recovery/recovery.controller.spec.ts

```diff
@@ -58,6 +63,8 @@ describe('Recovery (Unit)', () => {
     const moduleFixture: TestingModule = await Test.createTestingModule({
       imports: [AppModule.register(testConfiguration)],
     })
+      .overrideModule(JWT_CONFIGURATION_MODULE)
```

Comment:
As this opts in to `features.email`, the override is needed as`AuthGuard` is now on the `EmailController`.